Analytics/Classes/SEGAnalytics.m

```diff
@@ -164,10 +164,10 @@ - (void)_applicationWillEnterForeground {
     NSString *currentVersion = [[NSBundle mainBundle] infoDictionary][@"CFBundleShortVersionString"];
     NSString *currentBuild = [[NSBundle mainBundle] infoDictionary][@"CFBundleVersion"];
     [self track:@"Application Opened" properties:@{
-        @"from_background": @YES,
-        @"version" : currentVersion,
-        @"build" : currentBuild,
-    }];
+                                                   @"from_background": @YES,
```

Comment:
@tonyxiao I used Xcode - structure - re-indent. I'll try using spacecommander - that prateek mentioned.